### ISSUE TITLE
NewPipe

### ISSUE THREAD

**Issue by dschrempf** [micro-a5k/microg-unofficial-installer#8]
Hi,

I really enjoy your installer. Thank you very much!

I think that NewPipe should not be part of this package, because it is not MicroG related and I prefer to install it manually (if I want to use it).

Thanks,
Dominik

**Comment by theScrabi** [comment]
@ale5000-git honestly, I think @dschrempf is right.

**Comment by ale5000-git** [comment]
How I have already said many times there will be an option to exclude any single component so everyone will be happy.
But it may take a while to do it since I'm a bit busy.

**Comment by ale5000-git** [comment]
My installer remove YouTube that is pretty heavy especially on old phones, so I consider NewPipe really important as replacement.
The features that are missing in NewPipe compared to YouTube are irrelevant for me.

Now I have updated NewPipe and I have also added an option to do NOT install it (inside settings.conf) so anyone can choose.

In the future I will also add a script to update components like NewPipe and re-built the installer (so that anyone don't need to wait me to update it).
